CODE_OF_CONDUCT.md

```diff
@@ -109,19 +109,20 @@ most appropriate). If you would prefer not to do that, please feel free to
 report to the Code of Conduct Committee directly, or ask the Committee for
 advice, in confidence.
 
-You can report issues to the PySAL Code of Conduct committee, at
-***pysal-conduct@googlegroups.com***. Currently, the committee consists of:
+You can report issues to the PySAL Code of Conduct committee at
+***pysal-conduct@googlegroups.com***[create group and add link] or ***pysal/CoC***[create group and add link]. Currently, the committee consists of:
 
 - ***pysaler 1***
-- ***pysaler 2***
+- ***Stephanie Lumnitz***
```

Comment:
Hey @jGaboardi, thanks for adding me! :) 
My name is spelt: Stefanie. In the less known German version.

Reply:
So sorry! I thought I had the correct spelling there... I have made the correction.

Reply:
No problem, that happens quite often, thank you! Looks really really good!